portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -27,18 +27,21 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     UserService userService = UserServiceFactory.getUserService();
 
     List<String> list = new ArrayList<>();
```

Comment:
We should probably switch this to use a more informative name than "list".  Maybe something like "jsonElements"?  I'm not in love with that name so if you have a better idea that would be fine as well.